src/query/service/src/table_functions/show_grants/show_grants_table.rs

```diff
@@ -14,6 +14,7 @@
 
 use std::any::Any;
 use std::collections::HashMap;
+use std::collections::HashSet;
```

Comment:
Better use btree map, keep order

Reply:
HashMap is ok, we don't need keep id order